src/data/move.ts

```diff
@@ -4142,17 +4142,29 @@ export class IgnoreAccuracyAttr extends AddBattlerTagAttr {
   }
 }
 
-export class AlwaysCritsAttr extends AddBattlerTagAttr {
+export class AlwaysGetHitAttr extends AddBattlerTagAttr {
   constructor() {
-    super(BattlerTagType.ALWAYS_CRIT, true, false, 2);
+    super(BattlerTagType.ALWAYS_GET_HIT, true, false);
```

Comment:
```suggestion
    super(BattlerTagType.ALWAYS_GET_HIT, true, false, 0, 0, true);
```
This sets `lastHitOnly` to `true` for this attribute. PRE_MOVE battler tags tend to have weird overlapping issues when they come from multi-hit moves, and this change avoids those issues.